packages/office-ui-fabric-react/src/components/Persona/Persona.types.ts

```diff
@@ -139,6 +139,11 @@ export interface IPersonaSharedProps extends React.HTMLAttributes<PersonaBase> {
    * @deprecated Use `text` instead.
    */
   primaryText?: string;
+
+  /**
+   * Should the persona coin change to showing the initals instead of an image on hover
+   */
+  showInitialsOnHover?: boolean;
```

Comment:
Feels a little bit wrong to have `Persona` have this prop on it just to pass it over to `PersonaCoin`. We already have `coinProps` on it, but for some reason they are not of type `IPersonaCoinProps`. I wonder if this would be a good time to get that fixed.

Reply:
@Vitalius1's suggestion above makes sense and would be backwards compatible as far as I can tell since:

`IPersonaCoinProps extends IPersonaSharedProps extends React.HTMLAttributes<PersonaBase>`

So with the above suggestion we would refactor `coinProps` to be of type `IPersonaCoinProps`. Then `showInitialsOnHover` would be added to `IPersonaCoinProps` and read from that property bag in code.

```diff
diff --git a/packages/office-ui-fabric-react/src/components/Persona/Persona.types.ts b/packages/office-ui-fabric-react/src/components/Persona/Persona.types.ts
index 6e2cea7ce..197f5f601 100644
--- a/packages/office-ui-fabric-react/src/components/Persona/Persona.types.ts
+++ b/packages/office-ui-fabric-react/src/components/Persona/Persona.types.ts
@@ -127,7 +127,7 @@ export interface IPersonaSharedProps extends React.HTMLAttributes<PersonaBase> {
   /**
    * Optional HTML element props for Persona coin.
    */
-  coinProps?: React.HTMLAttributes<HTMLDivElement>;
+  coinProps?: IPersonaCoinProps;
 
   /**
    * Theme provided by High-Order Component.
```

@tabrumle do you agree? Would you be willing to make that change if it makes sense to you?

Reply:
Just finished looking over the code and planning on how best to make the changes. These are the steps I've come up with. Let me know if there's something I should tweak in my approach
1. Move showInitialsOnHover from IPersonaSharedProps to IPersonaCoinProps.
2. Change coinProps in IPersonaSharedProps to be of type IPersonaCoinProps.
3. Remove the spread operator that we call on coinProps in PersonaCoin.base.tsx. We can no longer do this because coinProps does not extend Div attributes anymore. (Could this break backwards compatibility?)
4. Add a prop to facepile that is something like showInitialsOnCoinHover?: boolean. The idea behind this boolean is to make the hover behavior taken care of by a single prop passed into Facepile.

Reply:
Pushed the changes talked about above. Talked with Kevin and decided to forgo doing step 3 to reduce the amount of Boolean flags on facepile.